### PR TITLE
Add datasource for each Loki instance

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: salt,logs

**Context**: 

**Summary**:

**Acceptance criteria**:

### CODE REVIEW

File: salt/metalk8s/addons/logging/loki/deployed/datasource.sls [hunk 1]
```diff
@@ -24,3 +33,11 @@ Deploy ConfigMap for Loki datasource:
...
               url: http://loki.metalk8s-logging.svc:3100/
               version: 1
+{%- for index in range(loki.spec.deployment.replicas) %}
+            - name: Loki-{{ index }}
+              uid: metalk8s-loki-{{ index }}
+              type: loki
+              access: proxy
+              url: http://loki-{{ index }}.metalk8s-logging.svc:3100/
+              version: 1
+{%- endfor %}
```
Comment (TeddyAndrieux) [inline salt/metalk8s/addons/logging/loki/deployed/datasource.sls:43]: Wondering if we can have one "default" datasource per type or if it's global, maybe we want to set the "Loki" datasource as default ?

**Review by TeddyAndrieux — COMMENTED** [review]
LGTM

Can you just add a changelog, and maybe we want to add a simple test just to be sure the "loki-x" service properly redirect to Loki
(I think about maybe one day a label in the Loki statefulset that change so the extra service we deploy does not get updated and we do not catch it)

File: salt/metalk8s/addons/logging/loki/deployed/datasource.sls (hunk 1 above)
Comment (gdemonet) [reply salt/metalk8s/addons/logging/loki/deployed/datasource.sls:43]: Not possible, we can only set one default per org

Comment (TeddyAndrieux) [reply salt/metalk8s/addons/logging/loki/deployed/datasource.sls:43]: :cry:

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,9 @@
 
 ### Enhancements
 
+- Add each Loki instance as datasources in Grafana dashboards
+  (PR[#3681](https://github.com/scality/metalk8s/pull/3681))
+
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: You will need to rebase since this should be in the 2.11.1 section (yet another issue with changelog as a simple file)

**Review by TeddyAndrieux — DISMISSED** [review]
Ok for me !

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**